src/store/ChartStore.js

```diff
@@ -451,6 +451,7 @@ class ChartStore {
         manageElement.textContent = t.translate('right-click to manage');
 
         if (this.state.isAnimationEnabled) animateChart(stxx, { stayPut: true }, tickAnimator);
+        stxx.chart.lockScroll = true;
```

Comment:
should this be `lockAutoScroll`?

Reply:
No, here it should be `lockScroll` to prevent ChartIQ from autoscrolling.